src/meta_schedule/space_generator/post_order_apply.cc

```diff
@@ -136,19 +136,31 @@ class PostOrderApplyNode : public SpaceGeneratorNode {
           stack.emplace_back(sch, blocks);
           continue;
         }
+
         Optional<String> ann = tir::GetAnn<String>(sch->GetSRef(block_rv), "schedule_rule");
-        if (ann.defined() == sch_rule.defined() || (ann.defined() && ann.value() == "None")) {
+        bool has_schedule_rule = ann.defined() && runtime::Registry::Get(ann.value()) != nullptr;
```

Comment:
Shall we store the value of `runtime::Registry::Get(ann.value())` so that we don't need to do that again later?

Reply:
done